tests/phpunit/integration/tests/Story_Post_Type.php

```diff
@@ -93,7 +96,12 @@ public static function wpSetUpBeforeClass( $factory ) {
 	public function set_up() {
 		parent::set_up();
 
-		$this->instance = $this->injector->make( \Google\Web_Stories\Story_Post_Type::class );
+		$experiments = $this->createMock( \Google\Web_Stories\Experiments::class );
+		$experiments->method( 'is_experiment_enabled' )
+					->willReturn( true );
```

Comment:
Hmm not sure I like basically enabling the experiment by default with this and then only having `test_get_has_archive_disabled_experiments` to test the _disabled_ scenario 🤔 

Need to review a bit more.

Reply:
There is only one place where we test for is the experiment is disabled and that is here. If we don't do this, then have to lots of repeated code to enabled it in the tests that need it.